app/server/controllers/UserController.js

```diff
@@ -640,6 +640,29 @@ UserController.resetPassword = function(token, password, callback){
   });
 };
 
+/**
+ * Logic for checking if the user's current location matches the hacks
+ * location so that the user can checkin
+ * @param {Function} callback [description]
+ */
+UserController.checkInByCurrentLocation = function(callback) {
+  callback();
+}
+
+/**
+ * Adds the schema's newly added fields to the records that do not contain it
+ * @param  {Function} callback [description]
+ */
+UserController.updateRecordsWithMissingFields = function(callback) {
+  var userSchema = User.schema.obj;
```

Comment:
Nice! This way we only need to update the value in one place, instead of searching through all the codebase